src/lib/contract/contract-comment.es6

```diff
@@ -0,0 +1,116 @@
+import {pad} from "../utils/string-utils";
+import CommentsGenerator from "../generators/comments-generator";
+
+const generator = new CommentsGenerator();
+
```

Comment:
You are using contract object in almost every method. Might be better if you organize this as class and accept contract object as constructor. Think objects and arrays are automatically passed by reference so there shouldn't be a problem